File: URI_1015.py
import math

linha1 = input().split(" ")
linha2 = input().split(" ")

x1, y1 = linha1
x2, y2 = linha2

distancia = math.sqrt((pow((float(x2) - float(x1)), 2) + pow((float(y2) - float(y1)), 2)))
print('%0.4f' % distancia)

File: URI_1015.java
import java.text.DecimalFormat;
import java.util.Scanner;

public class Main {

    public static void main(String[] args) {

        Scanner input = new Scanner(System.in);
        DecimalFormat df = new DecimalFormat("#.0000");

        double x1, y1, x2, y2;

        x1 = input.nextDouble();
        y1 = input.nextDouble();
        x2 = input.nextDouble();
        y2 = input.nextDouble();

        double distancia = Math.sqrt( Math.pow(x2-x1, 2) + Math.pow(y2 - y1, 2) );

        System.out.println(df.format(distancia));
    }
}
